script.recap/addon.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.recap" name="Recap" version="1.0.0" provider-name="robwebset">
+	<requires>
+		<import addon="xbmc.python" version="2.24.0"/>
+		<import addon="script.module.beautifulsoup" version="3.2.1"/>
+		<import addon="script.module.requests" version="2.3.0"/>
+	</requires>
+  	<extension point="xbmc.python.script" library="default.py"/>
+	<extension point="xbmc.service" library="service.py" start="login"/>
+	<extension point="kodi.context.item">
+		<menu id="kodi.core.main">
+			<item library="default.py">
+				<label>32001</label>
+				<visible>IsEmpty(Window(videos).Property("RecapHideContextMenu")) + [StringCompare(ListItem.dbtype,tvshow) | StringCompare(ListItem.dbtype,season) | StringCompare(ListItem.dbtype,episode)]</visible>
```

Comment:
This is also how Suitability, TvTunes and VideoExtras do it. It allows the user to configure if it is shown, otherwise I'd just always force them to be there - which didn't seem very nice (I believe giving the user the option is much better). Ideally it would read directly from the addon settings - but I could not see a way of doing that.